openhtf/__init__.py

```diff
@@ -37,11 +38,35 @@
 
 FLAGS = gflags.FLAGS
 FLAGS(sys.argv)
-
+gflags.DEFINE_string('log_dir', '/tmp', 'dir for log files')
+gflags.DEFINE_enum('log_output', 'console', ['console','file','both'], 'output option')
+gflags.DEFINE_enum('level', 'info', ['debug', 'info', 'warning', 'error', 'critical'], 'log level')
```

Comment:
Consider calling this something like ''log_level' instead of just 'level' for more clarity.